cnf-tests/testsuites/pkg/networks/network.go

```diff
@@ -10,14 +10,37 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	bondConfig = `
+	    "type": "bond",
+		"ifname": "%s",
+		"mode": "active-backup",
+		"failOverMac": 1,
+		"linksInContainer": true,
+		"miimon": "100",
+		"links": [ {"name": "%s"}, {"name": "%s"} ]
+`
+
+	macVlanConfig = `"type": "macvlan"`
+
+	hostLocalIpam = `"type": "host-local", "subnet": "%s/24"`
+
+	staticIpam = `"type":"static","addresses":[{"address":"%s/24"}]`
+)
+
+type token struct {
```

Comment:
What's token? If I am getting it right, we always add the ipam to the last item.
Would it make sense to have ipam as a seperate builder field?

Reply:
Done